### PR TITLE
Add code model link to psabi and add PIC model

### PR DESCRIPTION
**Pull request by kito-cheng**
We have add few document into RISC-V psABI, so put link here to prevent we need to repeat that here.

Also add PIC mode into this doc.

### CODE REVIEW

File: README.mkd [hunk 1]
```diff
@@ -161,6 +161,17 @@ within a single 2GiB address range, between the absolute addresses -2GiB and
 * `-mcmodel=medany`. The program and its statically defined symbols must lie 
 within a single 4GiB address range. `auipc` and `addi` pairs are used to 
 generate addresses.
+* Use any PIC or PIE option (e.g. -fpic, -fPIC, -fpie or -fPIE) will enable
```
Comment (asb) [inline README.mkd:164]: "Use any" => "Use of any"

File: README.mkd [hunk 2]
```diff
@@ -161,6 +161,17 @@ within a single 2GiB address range, between the absolute addresses -2GiB and
...
+medium position independent code model, this model is similar to the medium
```
Comment (asb) [inline README.mkd:165]: "medium" => "the medium"

I'd personally change ", this model" to ". This model" as well to reduce the length of the sentence.

**Review by asb — COMMENTED** [review]
Minor wording suggestions inline.

**Review by cmuellner — COMMENTED** [review]
Looks good after addressing Alex' comment.

**Comment by kito-cheng** [comment]
Update :)

**Review by cmuellner — APPROVED** [review]
LGTM

**Pull request merged**